app/_src/gateway/plugin-development/pdk/kong.service.request.md

```diff
@@ -218,8 +222,12 @@ Sets a header in the request to the Service with the given value.  Any existing
 **Parameters**
 
 * **header** (`string`):  The header name. Example: "X-Foo".
+{% if_version lte:3.5.x %}
 * **value** (`string|boolean|number`):  The header value. Example: "hello world".
-
+{% endif_version %}
+{% if_version gte:3.6.x %}
+* **of** (`array`):  strings|string|boolean|number value The header value. Example: "hello world".
```

Comment:
Looks like this didn't get generated correctly. Source is:

```
-- @tparam array of strings|string|number|boolean value The new value for the header.
```

Which _should_ generate into:

```
**value** (`array of strings|string|number|boolean`): The header value. Example: "hello world".
```
That is, there is now support for an array of strings, as well as non-array values.

Reply:
This applies to every instance of 
```
* **of** (`array`): strings|string|boolean|number value The header value. Example: "hello world".
```
in this PR